src/components/Header.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+// eslint-disable-next-line
+const Header = () => {
+  const headerStyle = {
+    padding: '20px 0',
+    lineHeight: '32px',
+  };
+
+  return (
+    <header style={headerStyle}>
+      <h1 style={{
+        fontSize: '60px',
+        fontWeight: '900',
+        marginBottom: '48px',
+        color: 'lightgrey',
+        textTransform: 'lowercase',
+        lineHeight: '48px',
+        textAlign: 'center',
```

Comment:
[OPTIONAL SUGGESTIONS]
Please consider using external CSS files instead of inline styling as it is the best practice. Click [here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) to see more HTML&CSS best practices